packages/client/tests/functional/issues/18276-batch-order/tests.ts

```diff
@@ -7,48 +7,53 @@ import type { PrismaClient } from './node_modules/@prisma/client'
 declare const newPrismaClient: NewPrismaClient<typeof PrismaClient>
 
 testMatrix.setupTestSuite(
-  () => {
-    test('executes batch queries in the right order when using extensions + middleware', async () => {
-      const prisma = newPrismaClient({
-        log: [{ emit: 'event', level: 'query' }],
-      }) as PrismaClient<{ log: [{ level: 'query'; emit: 'event' }] }>
+  ({ engineType }) => {
+    // TODO: does not receive the list of queries that is expected
```

Comment:
Yes, this is the same reason as to why tests fail elsewhere as well with `engineType = "wasm"`